src/campaigns/services/campaigns.service.ts

```diff
@@ -297,6 +346,23 @@ export class CampaignsService {
 
     return true
   }
+
+  private async sendCampaignLaunchEmail(user: User) {
+    try {
+      await this.emailService.sendTemplateEmail({
+        to: user.email,
+        subject: 'Full Suite of AI Campaign Tools Now Available',
+        // TODO: template is misspelled in Mailgun as well, must change there first.
+        template: 'campagin-launch',
+        variables: {
+          name: getFullName(user),
+          link: `${APP_BASE}/dashboard`,
+        },
+      })
+    } catch (e) {
+      this.logger.error('Error sending campaign launch email', e)
+    }
+  }
 }
 
 // async function claimExistingCampaignRequests(user, campaign) {
```

Comment:
we can delete this since we're not going to be migrating the Campaign Manager stuff